### ISSUE TITLE
Key with Object type and properties that cannot be validated

### ISSUE THREAD

**Issue by Sabst** [aldeed/meteor-simple-schema#510]
Hi,
If I define in a simple schema a key like this one (note the missing `blackbox: true` rule):
```
someKey: {
    type: Object
}
```
If I update the associated instance with a valid (but that cannot be validated) object (say {'a': 1}), the value **silently** stored in the DB is empty ({}).
Not only because it took me a while to discover the blackbox rule but also because a value is changed without notice, I think the validation error of an object should raise an explicit error rather than storing an empty value.

**Comment by judsonbsilva** [comment]
Me to.

**Comment by aldeed** [comment]
This is not due to simple-schema. I suspect you're probably using Collection2, which does object cleaning by default (removes disallowed keys). If you don't want them removed, then you can turn that off. See Collection2 docs.
